libs/common/src/test-helpers/convert-milliseconds-to-seconds.test.ts

```diff
@@ -0,0 +1,27 @@
+import { convertMillisecondsToSeconds } from './convert-milliseconds-to-seconds';
+
+describe('convert-milliseconds-to-seconds', () => {
+  describe('convertMillisecondsToSeconds', () => {
+    it('divides the passed in number by 1000 and rounds the result - rounding down', () => {
+      // Arrange
+      const milliseconds = 1729521390;
+
+      // Act
+      const seconds = convertMillisecondsToSeconds(milliseconds);
+
+      // Assert
+      expect(seconds).toEqual(1729521);
+    });
+
+    it('divides the passed in number by 1000 and rounds the result - rounding up', () => {
+      // Arrange
+      const milliseconds = 1729521500;
+
+      // Act
+      const seconds = convertMillisecondsToSeconds(milliseconds);
+
+      // Assert
+      expect(seconds).toEqual(1729522);
```

Comment:
Feels like we're kind of testing the native `Math.round` method here, I think just 1x assertion with `Math.round(milliseconds / 1000);` would suffice. But there's lots of conflicting opinions/preferences with unit tests so happy to approve.